scss/atoms/typography/_typography.scss

```diff
@@ -12,10 +12,10 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( max-width: $viewport-sm ) {
 		font-size: 16px;
 	}
-	@media ( max-width: 500px ) {
+	@media ( max-width: $viewport-xs ) {
```

Comment:
Since this PR is fixing breakpoints to be variables anyway, I would just set this to be mobile first as well while we're in there.  Otherwise you're just repeating work with two PRs.